bin/riemann-mongodb-serverstatus

```diff
@@ -11,9 +11,12 @@ class Riemann::Tools::Mongo
   opt :mongo_host, "Mongo hostname", :default => 'localhost'
   opt :mongo_port, "Mongo port", :default => 27017
   opt :mongo_db, "Mongo database", :default => 'local'
+  opt :mongo_user, "Mongo user for authentication", :default => ''
+  opt :mongo_pass, "Mongo password for authentication", :default => ''
 
   def initialize
   	@db = ::Mongo::MongoClient.new(opts[:mongo_host], opts[:mongo_port]).db(opts[:mongo_db])
+    @db.authenticate(opts[:mongo_user], opts[:mongo_pass]) unless opts[:mongo_user].empty? || opts[:mongo_pass].empty?
```

Comment:
Thats due to mixed tabs/spaces on the lines 18 and 20. If you really care, I can do another PR.